server/chat-plugins/scavengers.ts

```diff
@@ -1410,7 +1432,7 @@ const ScavengerCommands: Chat.ChatCommands = {
 		const res = ScavengerHunt.parseQuestions(params);
 		if (res.err) return this.errorReply(res.err);
 
-		room.game = new ScavengerHunt(room, user, hosts, gameType, res.result, mod);
+		room.game = new ScavengerHunt(room, user, hosts, gameType, res.result, isHTML, mod);
```

Comment:
The class is starting to have a bit too many arguments; could you make this a destructured parameter instead?